src/com/rakovets/course/practice/module3/Task02.java

```diff
@@ -22,5 +22,14 @@ public static void main(String[] args) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используяся переменные объявленные выше (их можно изменять)
         // Для проверки решения необходимо запустить @Test для данного class (в директории test)
+        if (hour >=0 && hour < 6) {
+            System.out.println("Good night");
+        }else if (hour >= 6 && hour < 12) {
+            System.out.println("Good morning");
+        }else if (hour >= 12 && hour < 18) {
+            System.out.println("Good day");
+        }else if (hour >=18 && hour <= 23) {

```

Comment:
Violated Code Convention
```suggestion
        } else if (hour >=18 && hour <= 23) {
```